plugins/main/public/controllers/management/components/management/cdblists/views/list-editor.tsx

```diff
@@ -368,6 +432,38 @@ class WzListEditor extends Component {
     );
   }
 
+  inputValidation({
+    field,
+    value,
+    onChange,
+    placeholder,
+  }: {
+    field: FieldTypes;
+    value: FieldStateTypes;
+    onChange: (e: any) => void;
+    placeholder: string;
+  }) {
+    return (
+      <EuiFormRow
+        fullWidth={true}
+        isInvalid={this.state.isInvalid.some(error => error.field === field)}
+        error={
+          this.state.isInvalid.filter(error => error.field === field)?.[0]
+            ?.message
+        }
```

Comment:
Extract the value to a constant within the current function.

Reply:
Done [310743b](https://github.com/wazuh/wazuh-dashboard-plugins/pull/7171/commits/310743b1942f73af2d2e2df3efd88ba0be294136)